packages/astro/src/prefetch/index.ts

```diff
@@ -104,7 +104,7 @@ function initHoverStrategy() {
 			clearTimeout(timeout);
 		}
 		timeout = setTimeout(() => {
-			prefetch(href, { with: 'fetch' });
+			prefetch(href, { with: 'fetch', ignoreSlowConnection: true });
```

Comment:
I think we shouldn't `ignoreSlowConnection` for hover and viewport strategies. That makes the slow connection detection never work?

The "fallback to tap strategy" should already be handled in the tap strategy iiuc.

Reply:
You are right. I think I misunderstood how strategy works with a link. Fixing it.